src/modules/ErrorHandler.js

```diff
@@ -0,0 +1,26 @@
+import store from '../state/store/configureStore'
+
+const errorHandler = (error) => {
+  if (error.response) {
+    if (error.response.data.data) {
+      if (error.response.data.data.errors) {
+        store.dispatch({
+          type: 'SET_ERROR_MESSAGE',
+          payload: error.response.data.data.errors,
+        })
+      }
+    } else if (error.response.data.error_message) {
+      store.dispatch({
+        type: 'SET_ERROR_MESSAGE',
+        payload: error.response.data.error_message,
+      })
+    }
+  } else {
+    store.dispatch({
+      type: 'SET_ERROR_MESSAGE',
+      payload: error.message,
+    })
+  }
+}
+
+export default errorHandler
```

Comment:
how about refactoring it into case statements. Will be easier to read and maintain

Reply:
Think it has to be nested - otherwise it will error out on that it can't find e.g. `errors of undefined`

Reply:
Not shure, we can start cases from `error.response.data.data.errors` then `error.response.data.data` etc. so other way around. Time is running out so I'll approve it, but I think this calls for refactoring chore.